### ISSUE TITLE
Varicella zoster virus infection after vaccination

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9618]
**Preferred term label:**
Varicella zoster virus infection after vaccination

**Synonyms**

**Definition (free text, please give PubMed ID)**
Infection with the varicella zoster  virus of the live-attenuated vaccine.
PMID:31527188

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
615401 
PMID: 19097825 1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
